data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/networkhandlers/PeersHandlerTest.java

```diff
@@ -59,11 +55,13 @@ public void shouldReturnArrayOfPeersIfPresent() throws Exception {
   }
 
   @Test
-  public void shouldReturnEmptyPeersArrayIfNoneConnected() throws Exception {
-    when(p2PNetwork.streamPeers()).thenReturn(Stream.empty());
-
+  public void shouldReturnEmptyPeersArrayIfNoneConnected(
+      @Mock Context mockContext, @Mock P2PNetwork<Peer> p2PNetwork) throws Exception {
```

Comment:
nit: i think i prefer the mocks defined at the top and not passed into each function...